catalyst-gateway/bin/src/util.rs

```diff
@@ -70,3 +80,82 @@ pub fn parse_child_assets(assets: &[MultiEraAsset]) -> Vec<Asset> {
         })
         .collect()
 }
+
+/// Eras before staking should be ignored
+pub fn valid_era(era: Era) -> bool {
+    matches!(
+        era,
+        Era::Shelley | Era::Allegra | Era::Mary | Era::Alonzo | Era::Babbage | Era::Conway
+    )
```

Comment:
Maybe change this to something like:

```suggestion
    !matches!(era, Era::Byron)
```

So when future eras come we don't ignore them?

Reply:
yea we're just ignoring byron for now.